### ISSUE TITLE
Resizing windows within rules?

### ISSUE THREAD

**Issue by pradyungn** [herbstluftwm/herbstluftwm#1266]
I've been looking through the documentation/googling all day, but I can't seem to find any evidence of being able to specify window geometry within rules. I know that in bspwm there are statements like this

```
bspc rule -a floating_term state=floating center=true rectangle=500x480+0+0
```
Is there any near-equivalent within hlwm?

**Comment by t-wissmann** [comment]
Do you want to specify the size when the window first appears or do you want to resize an existing window?

**Comment by tiosgz** [comment]
Hmm, isn’t this the same as #1157?

**Comment by t-wissmann** [comment]
If it's the first case, then I'd say so. (The other case is implemented by #1272 which I'd need anyway to test an implementation of a `floatgeometry` consequence)

**Comment by t-wissmann** [comment]
This has now been implemented in the current git version (PR #1311), so the rule you've requested translates into:
```bash
herbstclient rule class=floating_term floating=on floatplacement=center floating_geometry=500x480+0+0
```

**Comment by t-wissmann** [comment]
Since this issues is solved in the current git version, we can close this. If I missed anything, please open this issue again or feel free create another one.
